privacyidea/static/templates/header.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="">
     <meta name="author" content="Cornelius Kölbel" >
-    <link rel="icon" href="{{ instance }}/static/favicon.ico">
```

Comment:
Please delete the old favicon file.

Reply:
See https://github.com/privacyidea/privacyidea/commit/50c6c153f5a927299334c4337405b086644e0b6e